CMakeLists.txt

```diff
@@ -2,12 +2,25 @@ cmake_minimum_required(VERSION 2.8.12)
 PROJECT(facemask-plugin)
 math(EXPR BITS "8*${CMAKE_SIZEOF_VOID_P}")
 
+SET(PATH_DLIB "${CMAKE_CURRENT_SOURCE_DIR}/dlib")
+set (USE_AVX_INSTRUCTIONS TRUE)
+set (DLIB_NO_GUI_SUPPORT FALSE)
+set (DLIB_GIF_SUPPORT FALSE)
+set (DLIB_JPEG_SUPPORT FALSE)
+
+SET(PATH_FREETYPE "${CMAKE_CURRENT_SOURCE_DIR}/freetype2")
+
+if(TARGET libobs)
+	SET(PATH_OBS_STUDIO ${obs-studio_SOURCE_DIR})
```

Comment:
`${obs-studio_SOURCE_DIR}` evaluates to nothing during configure. If libobs is already defined as a target, you do not include "Findlibobs.cmake", instead you just directly link to `libobs`.

Reply:
This blocks building.